src/Form.jsx

```diff
@@ -880,7 +895,7 @@ function Form({
         );
         let submitPromise = null;
         if (featheryFields.length > 0)
-            submitPromise = client.submitStep(featheryFields);
+            submitPromise = client.submitStep(featheryFields, filePathMap);
```

Comment:
do we need to pass filePathMap directly from updateSessionValues since this is the same render and state may not have been updated yet?

Reply:
I didn't see a way for it to be in the same render since loading session values and submitting would definitely happen distinctly

Reply:
whoops didn't see that this was in submit - u right